traffic_ops/testing/api/v1/user_test.go

```diff
@@ -292,6 +294,26 @@ func GetTestUsers(t *testing.T) {
 	}
 }
 
+func GetTestAdminUsers(t *testing.T) {
+	adminUsers, _, err := TOSession.GetUsersByRole("admin")
+	if err != nil {
+		t.Errorf("cannot GET users by role: %v", err)
+	}
+	if *adminUsers[0].RoleName != "admin" {
```

Comment:
This can cause a segfault, need to check if the length of the returned slice is > 0 to ensure safe indexing, then need to make sure that the 0th item's `RoleName` property isn't `nil` to ensure safe dereferencing.

Reply:
done